docs/docsite/rst/network/dev_guide/developing_plugins_network.rst

```diff
@@ -0,0 +1,108 @@
+
```

Comment:
Would it make sense to include a reference example implementation that is bare bones and can be used a good start? The examples here are good, but it doesn't demonstrate everything end-to-end.